packages/orchestration/src/exos/ica-account-kit.js

```diff
@@ -128,29 +133,39 @@ export const prepareIcaAccountKit = (zone, { watch, asVow }) =>
         executeEncodedTx(msgs, opts) {
           return asVow(() => {
             const { connection } = this.state;
-            if (!connection) throw Fail`connection not available`;
+            if (!connection) {
+              throw Fail`Account not available or deactivated.`;
+            }
             return watch(
               E(connection).send(makeTxPacket(msgs, opts)),
               this.facets.parseTxPacketWatcher,
             );
           });
         },
-        /**
-         * Close the remote account
-         *
-         * @returns {Vow<void>}
-         * @throws {Error} if connection is not available or already closed
-         */
-        close() {
+        /** @type {HostOf<IcaAccount['deactivate']>} */
+        deactivate() {
           return asVow(() => {
-            /// TODO #9192 what should the behavior be here? and `onClose`?
-            // - retrieve assets?
-            // - revoke the port?
             const { connection } = this.state;
-            if (!connection) throw Fail`connection not available`;
+            if (!connection) throw Fail`Account not available or deactivated.`;
+            this.state.isInitiatingClose = true;
             return E(connection).close();
           });
         },
+        /** @type {HostOf<IcaAccount['reactivate']>} */
+        reactivate() {
+          return asVow(() => {
+            const { connection, port, requestedRemoteAddress } = this.state;
+            if (connection) {
+              throw Fail`Account is active. Call .deactivate() first.`;
```

Comment:
glad to see this explicitly handled. for the error message, why might somebody using `reactivate` want to deactivate first?  i.e. why not simply,
```suggestion
              throw Fail`Account is already active.`;
```